.goreleaser.yml

```diff
@@ -135,3 +135,19 @@ winget:
           owner: microsoft
           name: winget-pkgs
           branch: master
+
+cloudsmiths:
+  - organization: "task"
+    repository: "task"
+    formats:
+      - deb
+      - rpm
+    distributions:
```

Comment:
What about Alpine?

AFAIK `go-task` is published officially there, so maybe not needed, but asking anyway.

In theory would allow users to have a new release sooner.

Reply:
Yeah, I think we can add to test, and if anything we can just not document it.